readme.md

```diff
@@ -15,11 +15,17 @@ $ npm install --save dot-prop
 ```js
 var dotProp = require('dot-prop');
 
+// getter
 dotProp({foo: {bar: 'unicorn'}}, 'foo.bar');
 //=> 'unicorn'
 
-dotProp({foo: {bar: 'a'}}, 'foo.notDefined.deep');
```

Comment:
@stevemao I give'd you one https://github.com/lapwinglabs/rube/blob/1c8af1a0f48c9f27b048a2beadcf2df739605f42/lib/object.js#L85-L87

We have code like this in production:

```js
var translation = prop(i18n, 'some.keyset.key') || defaultValue;
```

We are perfectly aware, that some keys may be absent and want to get default value instead of `try...catching` errors. I think this is pretty valid use case.